src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -224,6 +224,7 @@ public MarkupResult ParseHtml(string html, FileAndType ft)
                     sources.Add(new LinkSourceInfo
                     {
                         Target = file,
+                        Anchor = anchor,
```

Comment:
this commit only add the anchor info into LinkSourceInfo, not add an attribute into the doc